http://cvs.gnome.org/viewcvs/glib/glib/gtimer.c?r1=1.29.2.3&r2=1.29.2.4
Fix a typo (read: build on G/FBSD)
===================================================================
RCS file: /cvs/gnome/glib/glib/gtimer.c,v
retrieving revision 1.29.2.3
retrieving revision 1.29.2.4
diff -u -r1.29.2.3 -r1.29.2.4
--- glib/gtimer.c	2006/10/01 05:51:15	1.29.2.3
+++ glib/gtimer.c	2006/10/06 18:44:33	1.29.2.4
@@ -257,7 +257,7 @@
 
   if (microseconds)
     *microseconds = (elapsed / 10) % 1000000;
-#elif HAVE_CLOCK_GETTIME
+#elif USE_CLOCK_GETTIME
   if (timer->active)
     GETTIME (timer->end);
 
